### PR TITLE
Added Azure Capability

### PR DESCRIPTION
**Pull request by JatinKinraAis**

### CODE REVIEW

**Comment by bgruening** [comment]
@JatinKinraAis thanks a bunch for your contribution! This really rocks!

File: django-cloudlaunch/baselaunch/admin.py [hunk 1]
```diff
@@ -109,6 +116,7 @@ class PublicServicesAdmin(admin.ModelAdmin):
 admin.site.register(models.AWS, CloudAdmin)
 admin.site.register(models.EC2, EC2Admin)
 admin.site.register(models.S3, S3Admin)
+admin.site.register(models.AZURE, CloudAdmin)
```
Comment (nuwang) [inline django-cloudlaunch/baselaunch/admin.py]: @JatinKinraAis Can you please make model name camelcase, like with OpenStack?

File: django-cloudlaunch/baselaunch/models.py [hunk 2]
```diff
@@ -102,6 +102,18 @@ class Meta:
         verbose_name_plural = "OpenStack"
 
 
+class AZURE(Cloud):
```
Comment (nuwang) [inline django-cloudlaunch/baselaunch/models.py]: @JatinKinraAis Please make model name conform to camel case

File: django-cloudlaunch/cloudlaunch/settings.py [hunk 3]
```diff
@@ -27,7 +27,7 @@
 SECRET_KEY = 'apsm2@#*opag-b7ae=&!w8r_q7077y%$wd@@0zfz1empu)r*!p'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False
+DEBUG = True
```
Comment (nuwang) [inline django-cloudlaunch/cloudlaunch/settings.py]: @JatinKinraAis This should probably remain False

File: django-cloudlaunch/cloudlaunch/settings.py [hunk 4]
```diff
@@ -139,13 +148,13 @@
 else:
     DATABASES = {
         'default': {
-            'ENGINE': 'django.db.backends.postgresql_psycopg2', # Add 'postgresql_psycopg2', 'mysql', 'sqlite3' or 'oracle'.
-            'NAME': 'cloudlaunch',                      # Or path to database file if using sqlite3.
+            'ENGINE': 'django.db.backends.sqlite3', # Add 'postgresql_psycopg2', 'mysql', 'sqlite3' or 'oracle'.
```
Comment (nuwang) [inline django-cloudlaunch/cloudlaunch/settings.py]: @JatinKinraAis These should probably be reverted too.

**Pull request merged**